contracts/core/Data/SportsAMMV2Data.sol

```diff
@@ -118,6 +125,49 @@ contract SportsAMMV2Data is Initializable, ProxyOwned, ProxyPausable {
         return _getTicketsData(ticketsArray);
     }
 
+    function getResolvedStatusForMarkets(
+        bytes32[] memory _gameIds,
+        uint16[] memory _typeIds,
+        uint16[] memory _playerIds
+    ) external view returns (bool[] memory resolvedMarkets) {
```

Comment:
Rename it to something that indicates boolean array (e.g. areMarketsResolved)

Reply:
done